foxpuppet/foxpuppet.py

```diff
@@ -8,7 +8,11 @@
 
 
 class FoxPuppet(object):
-    """ Class that sets up the api for interacting with the Firefox browser.
+    """
+        Class that sets up the interface for interacting with the Firefox
+        browser.
+
+        :param selenium: A selenium object
```

Comment:
I could use some help with the intersphinx mappings. For some reason it won't work and it appears that I am doing everything right